### PR TITLE
✨ Added Logos to storybook

### LINKED ISSUE #17: Logo Component
Create a Logo Component to use whenever needed

### PR DESCRIPTION
**Pull request by mrana17**
- Added 3 types of Logos as a story to storybook
- fixed all SVGs and removed the ones not needed
close #17

### CODE REVIEW

**Comment by SilviaBarcellini** [comment]
lgtm 👍

**Comment by ghost** [comment]
lgtm 👍

**Review by ghost — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
"Full Logo" competent don't work

**Review by RonjaMeinecke — APPROVED** [review]
👍

File: components/Logo/logo.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const Template: Story<LogoProps> = (args) => <Logo {...args} />;
+
+export const TextLogo = Template.bind({});
+TextLogo.args = {
+  src: "/text-logo.svg",
+};
+
+export const FahrerLogo = Template.bind({});
+FahrerLogo.args = {
```
Comment (m-wiemers) [inline components/Logo/logo.stories.tsx]: Don`t switch between english and german ;-)

```suggestion
export const DriverLogo = Template.bind({});
DriverLogo.args = {
```

**Review by m-wiemers — APPROVED** [review]
didn't know you were already a code king 👑

**Comment by mrana17** [comment]
> "Full Logo" competent don't work

**Comment by mrana17** [comment]
> "Full Logo" competent don't work

**Pull request merged**